build.gradle.kts

```diff
@@ -125,6 +129,18 @@ tasks {
     println("Source directories: ${sourceSets["main"].allSource.srcDirs}")
 }
 
+/**
+ * I can only get this to work if I use "mainClassName", even though
+ * the documentation says to use "mainClass". Without "mainClassName" I
+ * get an error that the main class is not set.
+ */
```

Comment:
@tcasstevens I'm going to remove this comment as it is no longer valid (we don't need mainClassName)

Reply:
Yes, please do